models/chainlink/avalanche_c/chainlink_avalanche_c_ccip_tokens_transferred_logs.sql

```diff
@@ -0,0 +1,60 @@
+{{
+  config(
+    alias='ccip_tokens_transferred_logs',
+    materialized='incremental',
+    file_format='delta',
+    incremental_strategy='merge',
+  )
+}}
+
+{% set incremental_interval = '7' %}
+
+SELECT
+  'avalanche_c' as blockchain,
+  block_hash,
+  contract_address,
+  data,
+  topic0,
+  topic1,
+  topic2,
+  topic3,
+  tx_hash,
+  block_number,
+  block_time,
+  index,
+  tx_index,
+  tx_from,
+  bytearray_to_uint256(bytearray_substring(data, 1, 32)) as total_tokens
+FROM
+  {{ source('avalanche_c', 'logs') }} logs
+WHERE
+  topic0 = 0x9f1ec8c880f76798e7b793325d625e9b60e4082a553c98f42b6cda368dd60008 -- Locked
+  AND block_time >= CAST('2023-07-06' AS date)
+  {% if is_incremental() %}
+            AND block_time >= date_trunc('day', now() - interval '{{incremental_interval}}' day)
```

Comment:
We can use the `incremental_predicate` here:
```suggestion
            AND {{ incremental_predicate('block_time') }}
```